pyiode/objects/equation.pyx

```diff
@@ -394,11 +394,12 @@ cdef class Equation:
         ... 
         ValueError: Cannot set LEC '(ACAF_ / VAF[-1]) := acaf2 * GOSF[-1] + acaf4 * (TIME=1995)' to the equation named 'ACAF'
         """
-        return self.c_equation.get_lec().decode()
+        return self.c_equation.get_lec().decode().replace('\n', '')
```

Comment:
Our users may have very long equations in their models. It is gonna be annoying for them if they have to often horizontally scroll if they want to see the whole LEC expression of a long equation. 

Option 1 (prefered): find how to modify `Equation.lec` and `Equation.__str__/__repr__` so that each "return carriage" appears as a newline line in the Python output.

Option 2: use the [textwrap.wrap](https://docs.python.org/3/library/textwrap.html#textwrap.wrap) function with a fixed value for the argument `width` (70? 100? 120?)